src/ripple/protocol/impl/STLedgerEntry.cpp

```diff
@@ -78,7 +78,7 @@ void STLedgerEntry::setSLEType ()
     type_ = mFormat->getType ();
     if (!setType (mFormat->elements))
     {
-        if (auto j = debugLog())
+        if (auto j = debugLog().error())
```

Comment:
@nbougalis You don't have to worry about the `Stream` lifetime relative to the `Journal`. Its the responsibility of the `Sink` to ensure that it outlives any of its journals or streams.